specs/latest/1.0/index.html

```diff
@@ -1671,6 +1703,7 @@ <h3><a name="WEBGLRENDERINGCONTEXT">The WebGL context</a></h3>
 
     const GLenum RGBA4                          = 0x8056;
     const GLenum RGB5_A1                        = 0x8057;
+    const GLenum RGBA8                          = 0x8058;
```

Comment:
Some WebGL conformance tests like conformance/context/constants-and-properties.html and conformance2/context/constants-and-properties-2.html (and maybe others) will have to be updated for this change.

Reply:
Yep, that's on the list of test changes that we'll need.

Reply:
@ccameron-chromium points out in https://chromium-review.googlesource.com/c/chromium/src/+/4887172 that it's necessary to add `RGB8` to WebGL 1.0 too, because `{alpha:false}` contexts use this as `drawingBufferFormat` even though `drawingBufferStorage` can't be called for such contexts afterward.